packages/bpk-component-accordion/src/BpkAccordionItem.scss

```diff
@@ -63,14 +63,14 @@
   }
 
   &__item-expand-icon {
-    @include bpk-themeable-property(fill, --bpk-accordion-color, $bpk-link-color);
+    @include bpk-themeable-property(fill, --bpk-accordion-color, $bpk-color-sky-blue-shade-03);
```

Comment:
This has changed due to link beinging used in multiple places and in the new brand is not the same colour